app/src/androidTest/java/org/wikipedia/tests/SearchEngineTests.kt

```diff
@@ -0,0 +1,81 @@
+package org.wikipedia.tests
+
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import junit.framework.Assert.assertEquals
+import junit.framework.Assert.assertTrue
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.wikipedia.main.MainActivity
+import org.wikipedia.pages.*
+
+@RunWith(AndroidJUnit4::class)
+class SearchEngineTests {
+    val explorePage = ExplorePage()
+    val searchBarPage = SearchBarPage()
+    val searchPage = SearchPage()
+    val navBarPage = NavBarPage()
+    val onboardingPage = OnboardingPage()
+    val articlePage = ArticlePage()
+
+    @Before
+    fun beforeTests() {
+        onboardingPage.tapOnSkipButton()
+    }
+
+    @Rule
+    @JvmField
+    var mActivityTestRule = ActivityScenarioRule(MainActivity::class.java)
+
+
+    @Test
+    fun searchBarTest() {
```

Comment:
Use more descriptive names for you tests. What does this test check? Anyone should be able to understand a goal of the tests by just looking at function name and not by looking at it's code. Reading a code this tests checks if there are results for a query.